src/op/op_util.cc

```diff
@@ -208,5 +208,31 @@ Stmt Substitute(Stmt s,
   return ir::Substitute(s, init);
 }
 
+Stmt TransformUpdate(const Stage& stage,
```

Comment:
This function is specific to Tensorize, move it to tensorize.cc, no need to keep comment in header file, move comment document to tensorize.cc as well

Reply:
done, move the function into tensorize.cc